environments/perftest.yml

```diff
@@ -45,4 +45,13 @@ cname:
     record: "a975c6f9-93cc-42fe-8cb1-c194eee0e2db.cloudapp.net"
   - name: "asverify.reformscan"
     ttl: 300
-    record: "asverify.reformscanperftest.blob.core.windows.net"
\ No newline at end of file
+    record: "asverify.reformscanperftest.blob.core.windows.net"
+  - name: "afdverify.idam-web-public"
+    ttl: 3600
+    record: "afdverify.hmcts-perftest.azurefd.net"
+  - name: "cdnverify.idam-web-public"
```

Comment:
I believe the `cdnverify` record is only used in Production.

Reply:
Makes sense @kremi 
@hmcts-devops can you confirm?

Reply:
hmcts-devops isn't a team, its just a bot user account.

cdnverify is only for prod (can be used in sandbox for testing).